courses/machine_learning/production_ml_systems/0_export_data_from_bq_to_gcs.ipynb

```diff
@@ -0,0 +1,228 @@
+{
```

Comment:
<p>Would it make sense to do something like this to create the dataset to handle errors (if it already exists) or perhaps too cumbersome?</p><p> </p><p> </p><p>from google.cloud import bigquery</p><p>bq = bigquery.Client(project = PROJECT)</p><p> </p><p>dataset = bigquery.Dataset(bq.dataset("taxifare"))</p><p>try:</p><p>&nbsp;&nbsp;bq.create_dataset(dataset) # will fail if dataset already exists</p><p>&nbsp;&nbsp;print("Dataset created")</p><p>except:</p><p>&nbsp;&nbsp;print("Dataset already exists")</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/584/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Done!</p>